### ISSUE TITLE
Low Contrast in UI is Hard to Read

### ISSUE THREAD

**Issue by jauyong** [GoogleForCreators/web-stories-wp#781]
## Feature Description

Some sections of the UI have low contrast making it hard to read. Possible accessibility issue?
![image](https://user-images.githubusercontent.com/5874851/77600193-81a31d80-6edd-11ea-8bf2-3cca641a0646.png)

Reported by Claudio

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by pbakaus** [comment]
Definitely an accessibility issue, thanks for calling it out. Could you take a look, @samitron7?

**Comment by swissspidy** [comment]
FWIW, the [storybook](https://google.github.io/web-stories-wp/) also points out color contrast issues. Even the [white-on-blue primary button](https://google.github.io/web-stories-wp/?path=/story/components-button--default)  (2.76:1 contrast ratio) doesn't meet WCAG recommendations (at least 3:1, ideally 4.5:1)

**Comment by samitron7** [comment]
I have these font set at FFF at 100%. I'm actually not sure where that screenshot came from.

**Comment by merapi** [comment]
> I have these font set at FFF at 100%. I'm actually not sure where that screenshot came from.

@samitron7 

Looking at https://stories-new-wordpress-amp.pantheonsite.io/wp-admin/post-new.php?post_type=web-story

`Add title` is a placeholder (if you have the title then you will see #FFF text).
`Images / Video` is a not active filter option (I believe it will be replaced with a dropdown in new Figma design)
`Primary button (white-on-blue)` contrast (tiny) issue remains?

Is this a thing for Alpha or should we revise it later? @pbakaus

**Comment by merapi** [comment]
One more update:
The title input will be also changed in the post-Alpha version (as per Figma).

_So I guess we can close this ticket (for Alpha)?_

**Comment by merapi** [comment]
Title and Filters will no longer look like this after Alpha - do we still want to adjust the colors now? @pbakaus

**Comment by pbakaus** [comment]
nope, no need to do redundant work.
